src/routes/create-klage.tsx

```diff
@@ -77,12 +80,21 @@ async function resumeOrCreateKlage(
         vedtakDate: null,
         userSaksnummer: null,
         internalSaksnummer: saksnummer,
-        fullmaktsgiver: fullmaktsgiver
+        fullmaktsgiver: fullmaktsgiver,
+        language: language
     });
 }
 
 const finneFullmaktsgiverError = (fnr: string) =>
     `Klarte ikke finne fullmaktsgiver med personnummer ${foedselsnrFormat(fnr)}.`;
 const oppretteKlageError = () => 'Klarte ikke opprette klage';
 
+export function getLanguage(query: queryString.ParsedQuery): Language {
+    const language = getQueryValue(query.language);
```

Comment:
Hvor settes `language` i query?

Reply:
> Hvor settes `language` i query?

Ah.. det var egentlig mest for å teste, men `language` skal kanskje kun komme fra dekoratøren?

Reply:
Dekoratøren setter språk i en cookie som vi ikke kan lese i frontend.
Jeg tror ikke vi kan se hvilket språk brukeren foretrekker i frontend i det hele tatt.